Readme.md

```diff
@@ -467,3 +493,114 @@ flowchart LR
+account should have the following roles in the project it runs:
+
+* CloudSQL Client (`roles/cloudsql.client)
+    * to be able to connect to the database running in the same project as Penelope application
+* Cloud Trace Agent
+    * to be able to connect to write traces to Google Monitoring
+
+### Backup
+
+The GCP Projects that are created for the pure purpose to store only backup data will be accessed by the ```backup```
+service accounts.
+
+#### permission for runner service account
+
+The ```runner``` service account should to be able to impersonate the```backup``` service account with the role: Service
+Account Token Creator (`roles/roles/iam.serviceAccountTokenCreator`)
+
+#### permission in data source
+
+The ```backup``` service account should have the following roles in the source project:
+
+* to be able to evaluate data store costs during backup creation
+    * Monitoring Viewer (`roles/monitoring.viewer`)
+* to be able to access project metadata
+    * `resourcemanager.projects.get`
+    * `resourcemanager.projects.list`
+* to be able to list and get GCS bucket metadata
+    * `storage.buckets.get`
+    * `storage.buckets.list`
+* to be able to make GCS Mirroring
+    * Private Logs Viewer  (`roles/logging.privateLogViewer`)
+* to be able to for GCS Mirroring strategy to check logging.list quota usage
+    * `serviceusage.services.list`
+* to be able to list and get BigQuery datasets/table/views metadata
+    * `bigquery.datasets.get`
+    * `bigquery.tables.get`
+    * `bigquery.tables.list`
+* to be able to list and export from BigQuery
+    * `bigquery.tables.createSnapshot`
+    * `bigquery.tables.export`
+    * `bigquery.tables.getData`
+    * `bigquery.tables.replicateData`
+
+#### permission in data sink
+
+The ```backup``` service account should have the following roles in the target (backup only) project:
+
+* to be able to manage GCS buckets and its objects
+    * `storage.buckets.create`
+    * `storage.buckets.delete`
+    * `storage.buckets.enableObjectRetention`
+    * `storage.buckets.get`
+    * `storage.buckets.getIamPolicy`
+    * `storage.buckets.list`
+    * `storage.buckets.setIamPolicy`
+    * `storage.buckets.update`
+    * `storage.objects.create`
+    * `storage.objects.delete`
+    * `storage.objects.get`
+    * `storage.objects.list`
+    * `storage.objects.update`
+* to be able to trigger export jobs in BigQuery from source project(s)
+    * BigQuery Job User (`roles/bigquery.jobUser`)
+* to be able to create&update Storage Transfer jobs
+    * Storage Transfer User (`roles/storagetransfer.user`)
+* to be able to clean up backups that transit to status `BackupDeleted`
+    * `storagetransfer.jobs.delete`
+    * `storagetransfer.jobs.get`
+    * `storagetransfer.jobs.list`
+    * `bigquery.jobs.delete`
+    * `bigquery.jobs.get`
+    * `bigquery.jobs.list`
+* to be able to check compliance permissions
+    * `iam.denypolicies.list`
+    * `iam.denypolicies.get`
```

Comment:
![image](https://github.com/ottogroup/penelope/assets/5013103/73affbb7-33e7-4fa9-bcde-f41cc767c9f5)
please add an image to the documentation